modules/mcaptcha.nix

```diff
@@ -262,11 +262,14 @@ in {
       ensureUsers = [
         {
           name = cfg.user;
-          ensurePermissions = {"DATABASE ${cfg.settings.database.name}" = "ALL PRIVILEGES";};
         }
       ];
     };
 
+    systemd.services.postgresql.postStart = lib.mkIf cfg.database.createLocally (lib.mkAfter ''
+      $PSQL ${cfg.settings.database.name} -c 'GRANT ALL PRIVILEGES ON SCHEMA public TO "${cfg.user}"'
```

Comment:
https://nixos.org/manual/nixos/unstable/#module-services-postgres-initializing-extra-permissions

Reply:
Thanks for that pointer!